airbyte-cdk/python/airbyte_cdk/sources/streams/core.py

```diff
@@ -502,3 +505,11 @@ def _checkpoint_state(  # type: ignore  # ignoring typing for ConnectorStateMana
         #  to reduce changes right now and this would span concurrent as well
         state_manager.update_state_for_stream(self.name, self.namespace, stream_state)
         return state_manager.create_state_message(self.name, self.namespace)
+
+    @property
+    def configured_json_schema(self) -> Optional[Dict[str, Any]]:
```

Comment:
nit: can you add a warning specifying this is set from the read method, meaning the configured_json_schema will always be done for discovers?

Reply:
added